io.github.bcpierce00.unison.metainfo.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.bcpierce00.unison</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>
+    <!-- unison -->GPL-3.0-or-later AND LGPL-2.0-only AND LGPL-2.1-only AND LGPL-2.1-or-later <!-- camlp-streams, lablgtk3 --> AND (LGPL-2.1-only WITH OCaml-LGPL-linking-exception) <!-- ocaml-cairo --> AND (LGPL-3.0-or-later WITH OCaml-LGPL-linking-exception)</project_license>
```

Comment:
it's the license of you app not every component ever in the manifest or not sure what this is?

Reply:
okay i just copied license stuff from Fedora spec files for everything bundled into the Flatpak, if it's just the "effective" license of the application it can be simplified of course :)